File: js/tests/findTheBomb.test.js
const bomb = require("../findTheBomb");

const data = [
  ["There is a bomb.", "Duck!!!"],
  ["Hey, did you find it?", "There is no bomb, relax."],
  ["Hey, did you think ther is a bomb?", "Duck!!!"],
  ["This goes boom!!!", "There is no bomb, relax."],
  ["Hey, did you find the BoMb?", "Duck!!!"]
];

describe("bomb", () => {
  describe("Return message if string contains the word bomb", () => {
    test.each(data)("bomb(%p) = %p", (str, result) => {
      let actual = bomb(str);

      expect(actual).toEqual(result);
    });
  });
});

File: CSharp/Tests/FindTheBombTest.cs
using Xunit;

namespace CSharp.Tests
{
    public class FindTheBombTest
    {
        [Theory]
        [InlineData("There is a bomb.", "Duck!!!")]
        [InlineData("Hey, did you find it?", "There is no bomb, relax.")]
        [InlineData("Hey, did you think there is a bomb?", "Duck!!!")]
        [InlineData("This goes boom!!!", "There is no bomb, relax.")]
        [InlineData("Hey, did you find the BoMb?", "Duck!!!")]
        [InlineData("Commotion in the third, a bomb is found!", "Duck!!!")]
        public void Bomb_StringInput_ReturnMessageIfStringHasWordBomb(string txt, string expected)
        {
            var actual = FindTheBomb.Bomb(txt);

            Assert.Equal(expected, actual);
        }
    }
}